tests/integration/upgrade_test.go

```diff
@@ -57,7 +61,7 @@ func (s *UpgradeSuite) SetupSuite() {
 	s.namespace = "upgrade-ns"
 	useDevices := true
 
-	s.op, s.k8sh = StartTestCluster(s.T, s.namespace, "bluestore", false, useDevices, 1, installer.Version0_8)
+	s.op, s.k8sh = StartTestCluster(s.T, s.namespace, "bluestore", false, useDevices, 3, installer.Version0_8)
```

Comment:
This only starts one OSD in most test envs, right?
Will we implement multiple OSDs for the tests (/this specific test) later on?

Reply:
right, we only start one filestore osd. However, if devices are available, we will consume all devices. But we need to make this smarter to allow multiple OSDs. I'll open an issue for that.